### ISSUE TITLE
Plant Model page does not currently load for some reason

### ISSUE THREAD

**Issue by cshenry** [ModelSEED/PlantSEED#8]
I clicked on the "View model" link on a plant genome and got here:

http://140.221.67.62/#/model/plantseed/Models/Alyrata-v.1.0

But I was left with an empty page...

**Comment by nconrad** [comment]
yeah, the links use whatever is available at /plantseed/Models (http://140.221.67.62/#/data/plantseed/Models).  I think Sam needs to build the models.  The only thing model currently there is a hidden .Athaliana-TAIR10_Model .

**Comment by samseaver** [comment]
Yes, I was testing/playing with Athaliana, I'll do the others tonight.

**Comment by samseaver** [comment]
All PlantSEED models reconstructed and gapfilled. Model links appear to be working.

**Comment by nconrad** [comment]
The 'view model' links should be fixed and deployed.   We should reduce the size of the plantseed images if we have time.

**Comment by cshenry** [comment]
Hey guys,

On the PlantSEED model viewer, the gene links are broken. For PlantSEED models, the gene links need to go to our feature page� not PATRIC.

Chris

**Comment by cshenry** [comment]
Also, I understand that for public models, we can�t offer a �runfba� or �gapfill� button, but can we have a �Copy model� button? This would copy the model to my own workspace? Then I would see the model on the �My models� menu, and I could runfba or gap filling.

**Comment by cshenry** [comment]
This might be nice to have on the genomes page too. Like right below the �view model� link, could I have a �copy genome� link, which would copy the genome to my workspace.

We could add �copy_model� and �copy_genome� commands to the API. What do we think?

Chris

**Comment by nconrad** [comment]
Fine with me.  Since there are hidden directories involved (right?), the api calls would be good to have.

**Comment by nconrad** [comment]
oh, it'd also be nice if the copy_model can return the same structure as list_models.

**Comment by cshenry** [comment]
Exactly.

**Comment by mmundy42** [comment]
Is copy_genome really needed?  The genome is already being copied by ModelReconstruction().

**Comment by cshenry** [comment]
It is for PlantSEED, because PlantSEED is going to permit the modification of genome annotations� thus, genomes have to be owned by users�

In the long run� I think we are going to merge the concept of genome and model together in ModelSEED/PlantSEED. For now though, they will be separate objects.
